### ISSUE TITLE
Built CSS doesn't save to specified output directory

### ISSUE THREAD

**Issue by chadian** [isitedesign/Phoenix#116]
Running latest version of Phoenix @ 412139514bfce9f622c98dba8034b11e96e0a271 and specifying the output path doesn't move the files to the expected output path.

Example driver for css within gulpfile.js:
```
			driver: 'css',
			paths: ['./css/*.scss'],
			base: './css',
			watchPaths: ['css/**'],
			autoClean: true,
			autoCleanPaths: ['./css/*.css'],
			output: './something-else/',
```

It looks like everything is fine up to `var stream = through2.obj(saveFile);` in `Assets/src/node_modules/gulp/node_modules/vinyl-fs/lib/dest/index.js` but possibly fails within the `saveFile` callback because the file insists on saving to `../dist/`.

Maybe, I'm missing something in terms of setup of the configuration values?

**Comment by elseloop** [comment]
@chadian Was there ever any discussion around this? Is it still an issue?

**Comment by chadian** [comment]
Looks like it's still an issue.

**Comment by ajmueller** [comment]
I'm digging the response time for each comment.  See you guys in 6 months!

**Comment by ajmueller** [comment]
@devinrayolsen maybe this is related to the Jade template output issue?

**Comment by devinrayolsen** [comment]
@ajmueller Correct. I just took the comments off that other ticket cause I realized it's not any config option I can change (and distracts from the conversation). 

In my case, old PHP driver is not a driver at all, so files get copied over under all builds correctly. Once you use a driver for the task, the output option will no longer work correctly (that's what I'm experiencing at least).
